EpikCord/channels.py

```diff
@@ -496,23 +496,28 @@ async def handle_close(self):
         self._closed = True
         if self.ws.close_close == GatewayCECode.UnknownOpcode:
             raise ClosedWebSocketConnection(
-                "EpikCord has sent an invalid OpCode to the Voice WebSocket. Report this at https://github.com/EpikCord/EpikCord.py/issues"
+                "EpikCord has sent an invalid OpCode to the Voice WebSocket."
+                " Report this at https://github.com/EpikCord/EpikCord.py/issues"
             )
         elif self.ws.close_code == GatewayCECode.DecodeError:
             raise ClosedWebSocketConnection(
-                "EpikCord has sent an invalid identify to the Voice WebSocket. Report this at https://github.com/EpikCord/EpikCord.py/issues"
+                "EpikCord has sent an invalid identify to the Voice WebSocket."
```

Comment:
Missing commas?

Reply:
no, python compiler will merge the string together :)